adapters/handlers/rest/handlers_schema.go

```diff
@@ -321,19 +327,8 @@ func (s *schemaHandlers) getTenant(
 				WithPayload(errPayloadFromSingleErr(err))
 		}
 	}
-	if len(tenants) == 0 {
-		s.metricRequestsTotal.logUserError(params.ClassName)
```

Comment:
the metric should probably stay?

Reply:
Ah no, there is a metric log of the error on L313 but this metric doesn't make sense anymore since `tenants` is renamed to `tenant` with type `*models.TenantResponse`. I suppose we could have a check of whether `tenant` is `nil` here and log that if so?

Reply:
that sounds right :)